api/v1alpha1/oidc_types.go

```diff
@@ -62,6 +62,13 @@ type OIDC struct {
 	// If not specified, uses the default redirect URI "%REQ(x-forwarded-proto)%://%REQ(:authority)%/oauth2/callback"
 	RedirectURL *string `json:"redirectURL,omitempty"`
 
+	// Any request that matches any of the provided matchers won’t be redirected to OAuth server when tokens are not valid.
+	// Automatic access token refresh will be performed for these requests, if enabled.
```

Comment:
This comment is somewhat ambiguous.  The request can be denied with a 401 response in two conditions:
* User has logged in but tokens are expired.
* User has not logged in, and no tokens in the cookie.

Reply:
Agreed. Have changed this comment to 

```Any request that matches any of the provided matchers (with either tokens that are expired or missing tokens) will not be redirected to the OIDC Provider.```